README.md

```diff
@@ -14,3 +14,14 @@ The following glide commands can be used to update the dependencies of this proj
 
     glide update --strip-vendor
     glide-vc --use-lock-file --no-tests --only-code
+
+## Deployment
+
+The `master` branch of this repository is configured for autobuilding on Quay.
+The resulting image is pushed to `quay.io/coreosinc/coreos-userdata-validator:latest`,
+which is not available for pulling by the general audience.
+
+The `latest` tag of this image is deployed on CoreOS kubernetes cluster
+via the relevant Helm [chart], with an appropriate pull-secret.
+
+[chart]: https://github.com/coreos-inc/infra-kubernetes/tree/master/charts/userdata-validator
```

Comment:
We probably shouldn't link to a private repo from this public one.

Reply:
Indeed, removed.